management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -241,7 +242,7 @@ public void test_stats_collection() throws Exception {
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
     assertThat(stats[0].getStatistic(Counter.class, "GetCounter").getValue(), equalTo(4L));
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
-    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, equalTo(1));
+    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, greaterThanOrEqualTo(1));
```

Comment:
Also it is because we are just interested in the exact next stat that has been computed. Not the eventual extra one. The other one can or cannot be there, depending if the scheduler had some time to compute it. Anyway, we do not care about this other one. Only the first next one.